Add Hue White Ambiance Filament Standard

Midbin opened the pull request:
This Pull Request adds Support for the Hue White Ambiance Filament Standard bulb. Measurement happened with 3 bulbs and a Shelly 1PM.

bramstroker commented:
Hi, thanks for contributing.

See the github action validation. There is an issue at line 314. It reads `1,210,0,27`, but the power must have a period as decimal separator not a comma.
Did you do manual transformations in the file? Because I don't see how the measure tool could generate this. Also the lines are orders by mired value now and not by brightness. Did you do that yourself?

Midbin commented:
Hallo, 

I changed the ordering in the script from checking all the mired levels at a given brightness to checking first all brightness levels at a given mired. 
My Measurement always got stuck when the wattage did not change (which happens quite often when the brightness stays the same) and required a nudge, which added a lot of time to measuring. After changing the brightness on every step that did not happen nearly as often, and my measuring device only needed 2 or 3 nudges total. 

Because I changed the ordering there was no additional waiting time for a significant brightness change, and I had two occasions where there was something like 3 watts for brightness 1. So I remeasured these manually and changed them in the csv. I must have entered a comma out of habit in one location.

Midbin commented:
I am also working on a pull request for a setting to change the ordering, if other persons do have the same problem as me, but I am still testing that.

bramstroker commented:
Ah ok thanks for clarifying. And cool you are working on a PR for this alternative method of measuring. I can see how this can be useful for some measure devices which don't update frequently enough on minor changes.
